src/metabase/pulse/render/body.clj

```diff
@@ -325,19 +326,17 @@
                          "timeseries"
                          "ordinal")]
     (merge
-     {:colors      (public-settings/application-colors)
-      :stacking    (if (:stackable.stack_type viz-settings) "stack" "none")
-      :show_values (boolean (:graph.show_values viz-settings))
-      :x           {:type   (or (:graph.x_axis.scale viz-settings) default-x-type)
-                    :format x-format}
-      :y           {:type   (or (:graph.y_axis.scale viz-settings) "linear")
-                    :format y-format}
-      :labels      labels}
+     {:colors                 (public-settings/application-colors)
+      :stacking               (if (:stackable.stack_type viz-settings) "stack" "none")
+      :x                      {:type   (or (:graph.x_axis.scale viz-settings) default-x-type)
+                               :format x-format}
+      :y                      {:type   (or (:graph.y_axis.scale viz-settings) "linear")
+                               :format y-format}
+      :labels                 labels
+      :visualization_settings (or viz-settings {})}
```

Comment:
I just pass the whole `visualization_settings`, because it seems weird that we need to hand-pick which settings we want the static viz component to see. And it was hard to extend static viz when we need to read some other existing settings that we haven't read before.

So `:show_values` and `:series_settings` are removed in favor of this new `:visualization_settings`.

One thing to note that's there a test failed because the visualization settings are undefined. Since I'm short on time, I think the safer way to get around this is to always default it to an empty object.

This is the test that failed (I couldn't find the check that failed now)
https://github.com/metabase/metabase/blob/53326d747ac749be9ecaa4eb10fb0baab52cf2e3/test/metabase/pulse/render_test.clj#L25-L31